tests/Integration/Requests/ExecuteTest.php

```diff
@@ -47,7 +64,7 @@ public function testExecuteInsertsRows() : void
 
     public function testExecuteFetchesAllRows() : void
     {
-        $response = $this->client->execute('SELECT * FROM exec_query');
+        $response = $this->client->execute(self::provideScanQuery());
```

Comment:
Let's replace all `provideScanQuery()` calls with something like this:

```suggestion
        $seqScanOrEmpty = $this->tarantoolVersionSatisfies('>=2.11') ? 'SEQSCAN' : '';
        $response = $this->client->execute("SELECT * FROM $seqScanOrEmpty exec_query");
```

Btw, `>=2.11.0-RC1` constraint will most likely work too, but I haven't tested it.

Reply:
Updated to use ternary expression, separate variable for `seqScan` and check with `tarantoolVersionSatisfies`.